stellarphot/photometry/tests/test_photometry.py

```diff
@@ -39,77 +39,107 @@
 SEED = 5432985
 
 
-# The default coordinate system and shift tolerance to use for the tests
-COORDS2USE = "pixel"
-SHIFT_TOLERANCE = 6
-DEFAULT_SOURCE_LOCATIONS = SourceLocationSettings(
-    source_list_file="",
-    shift_tolerance=SHIFT_TOLERANCE,
-    use_coordinates=COORDS2USE,
-)
+@pytest.fixture
+def source_locations():
+    return SourceLocationSettings(
+        source_list_file="",
+        shift_tolerance=6,
+        use_coordinates="pixel",
+    )
 
-PHOTOMETRY_OPTIONS = PhotometryOptionalSettings()
-# This used to be the default; it has switched to exact
-PHOTOMETRY_OPTIONS.method = "center"
-
-# A camera with not unreasonable settings
-FAKE_CAMERA = Camera(
-    data_unit=u.adu,
-    gain=1.0 * u.electron / u.adu,
-    name="test camera",
-    read_noise=0 * u.electron,
-    dark_current=0.1 * u.electron / u.second,
-    pixel_scale=1 * u.arcsec / u.pixel,
-    max_data_value=40000 * u.adu,
-)
 
-# Camera with no read noise or dark current
-ZERO_CAMERA = Camera(
-    data_unit=u.adu,
-    gain=1.0 * u.electron / u.adu,
-    name="test camera",
-    read_noise=0 * u.electron,
-    dark_current=0.0 * u.electron / u.second,
-    pixel_scale=1 * u.arcsec / u.pixel,
-    max_data_value=40000 * u.adu,
-)
+@pytest.fixture
+def photometry_optional_settings():
+    pos = PhotometryOptionalSettings()
+    # This used to be the default; it has switched to exact
+    pos.method = "center"
+    return pos
+
+
+@pytest.fixture
+def fake_camera():
+    # A camera with not unreasonable settings
+    return Camera(
+        data_unit=u.adu,
+        gain=1.0 * u.electron / u.adu,
+        name="test camera",
+        read_noise=0 * u.electron,
+        dark_current=0.1 * u.electron / u.second,
+        pixel_scale=1 * u.arcsec / u.pixel,
+        max_data_value=40000 * u.adu,
+    )
+
+
+@pytest.fixture
+def zero_camera():
+    # Camera with no read noise or dark current
+    return Camera(
+        data_unit=u.adu,
+        gain=1.0 * u.electron / u.adu,
+        name="test camera",
+        read_noise=0 * u.electron,
+        dark_current=0.0 * u.electron / u.second,
+        pixel_scale=1 * u.arcsec / u.pixel,
+        max_data_value=40000 * u.adu,
+    )
+
+
+@pytest.fixture
+def fake_obs():
+    # Fake observatory location
+    return Observatory(
+        name="test observatory",
+        latitude=0 * u.deg,
+        longitude=0 * u.deg,
+        elevation=0 * u.m,
+    )
 
-# Fake observatory location
-FAKE_OBS = Observatory(
-    name="test observatory", latitude=0 * u.deg, longitude=0 * u.deg, elevation=0 * u.m
-)
 
 # The fake image used for testing
```

Comment:
Did you forget to migrate `FAKE_CCD_IMAGE` to be within a fixture or does it not work that way?

Reply:
That seemed OK for now since we have it in a function, but I can fixture-ize it if you want.

Reply:
No need, just checking.